demos/embeddings/watch-your-step-cora-demo.ipynb

```diff
@@ -11,9 +11,11 @@
     "from stellargraph.layer.watch_your_step import WatchYourStep, get_embeddings\n",
```

Comment:
<p>We don't do this in any other notebook. Is it important to do it here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1031/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I think that's left over code from development when I messed up the imports, I'll change it to `from stellargraph.layer import WatchYourStep, get_embeddings`

Reply:
I think you're looking at the wrong place. ReviewNB always attaches the comment to the first line of the diff, not the actual cell that it was commented on. You have to look on ReviewNB itself to see the right place. In this case, it's attached to the `tf.random.set_seed` cell.

Reply:
ah gotcha, just thought it would be nice for the notebook to reproducible but its not important here.

Reply:
Yeah, we seemingly don't care about the notebooks being exactly reproducible, just that rerunning them gives similar results.